### ISSUE TITLE
Task: Ruleset expander: move buttons

### ISSUE THREAD

**Issue by Jay-o-Way** [U-C-S/Hurl#113]
Easy to do. This is current concept:

![image](https://github.com/U-C-S/Hurl/assets/65828559/8f9d78d0-e468-463a-96aa-4b47b4c4d490)

With the current MinWidth, the text is quickly trimmed. We _could_ choose to combine these buttons* under a <kbd>●●●</kbd> button, but that means more clicks = more RSI. Increasing the MinWidth to 500 is common and removing the Grid Columns is simple.

\* Related:
- removed the "3 rules found" text - _maybe_ use an InfoBadge for the number later
- we could use an icon (in stead of text) for the browser

If possible, it would be nice if these expanders could be dragged to order, but I am not sure if that is even possible.

**Comment by U-C-S** [comment]
> the text is quickly trimmed. We could choose to combine these buttons* under a ●●● button

What do you think about moving the delete and edit btns into the expanded body ?

---

> It would be nice if these expanders could be dragged to order, but I am not sure if that is even possible

I spent some time doing this, but i couldn't get it working correctly. I want to come back to this later on, once other stuff is done.

> we could use an icon (in stead of text) for the browser

We can do that. I want to see and change how icons are handled currently. Maybe after that.

**Comment by Jay-o-Way** [comment]
> What do you think about moving the delete and edit btns into the expanded body ?

Hm, that's 50% of the idea. At the moment i think it would be nice to have these buttons appear at a "hover" event. But I haven't worked with that yet.
